docker/start-app.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This configuration is a temporary workaround until we migrate to CCF 4.0.3, which supports
+# a cchost "config-timeout" option to accomplish a similar result. This logic is required when
+# the SCITT container is deployed in Kubernetes without any orchestration sidecars.
+
+WAIT_TIME_SEC=3
+TIMEOUT_SEC=300
+
+echo "Starting app"
+args=$(printf '%s ' "$@") # concatenate all input cchost arguments into a single string
```

Comment:
should not be necessary, using `"$@"` subsequently would work

Reply:
Done